New ESP32 Framework Kills IPAddress Check

Using an updated [arduino-esp32](https://github.com/espressif/arduino-esp32) build the folowing errors were raised:

```
.pio\libdeps\tft_espi\LCBUrl\src\LCBUrl.cpp:301:23: error: ambiguous overload for 'operator==' (operand types are 'IPAddress' and 'u32_t {aka unsigned int}')
         if (ipaddress == IPADDR_NONE)
```

Apparently `IPADDR_NONE` is now a `u32_t`.

--- a/src/LCBUrl.cpp
+++ b/src/LCBUrl.cpp
@@ -298,7 +298,7 @@ String LCBUrl::getIPAuthority() // Returns {username (optional)}:{password (opti
         {
             ipauthority.concat(F("@"));
         }
-        if (ipaddress == INADDR_NONE)
+        if (ipaddress == (IPAddress)INADDR_NONE)
         {
             ipauthority.concat(getIP(getHost().c_str()).toString());
         }

